### PR TITLE
Add offset, polar, and interpolate to Point class

### PR DESCRIPTION
**Pull request by jschatz1**
This adds three point methods. I did not see any `Point` specs so I did not add tests for them. I did run them against the AS3 `Point` library and got the same results.

### CODE REVIEW

File: src/easeljs/geom/Point.js [hunk 1]
```diff
@@ -99,6 +99,53 @@ this.createjs = this.createjs||{};
 		return this;
 	};
 	
+	/** 
+	 * Offsets the Point object by the specified amount. The value of dx is added to the original value of x to create the new x value. The value of dy is added to the original value of y to create the new y value.
+	 * @method offset
+	 * @param {Number} The amount by which to offset the horizontal coordinate, x.
```
Comment (lannymcnie) [inline src/easeljs/geom/Point.js]: All the parameters are missing names. If you compile the docs, the first word after the type will be the parameter name. The correct syntax is:
`@param {type} name Description`
OR
`@param name {type} Description`.

Note you can also specify optional params, and default values:
`@param [type] name Optional Parameter`
`@param [type=value] name Optional parameter with default value]`

http://yui.github.io/yuidoc/syntax/

File: src/easeljs/geom/Point.js [hunk 2]
```diff
@@ -99,6 +99,53 @@ this.createjs = this.createjs||{};
...
+	Point.polar = function(len, angle, pt) {
+		pt = pt||{};
+		pt.x = len * (Math.cos(angle));
+		pt.y = len * (Math.sin(angle));
+		return pt;
+	};
+
+	/** 
+	 * Determines a point between two specified points. The parameter `f` determines where the new interpolated point is located relative to the two end points specified by parameters `pt1` and `pt2`. The closer the value of the parameter `f` is to 1.0, the closer the interpolated point is to the first point (parameter `pt1`). The closer the value of the parameter `f` is to 0, the closer the interpolated point is to the second point (parameter `pt2`).
+	 * @method offset
```
Comment (lannymcnie) [inline src/easeljs/geom/Point.js]: Wrong method name.

**Comment by lannymcnie** [comment]
Added a few comments on the documentation.

**Comment by jschatz1** [comment]
@lannymcnie Thanks I'll fix that up.

**Comment by jschatz1** [comment]
@lannymcnie Fixed those things up. For optional params I followed the styles you have in [Matrix2D.js](https://github.com/CreateJS/EaselJS/blob/master/src/easeljs/geom/Matrix2D.js#L468).

**Comment by lannymcnie** [comment]
Thanks! Will chat internally about the additions, and merge them.

**Comment by jschatz1** [comment]
@lannymcnie I guess you guys aren't doing this as this has been stale for a while? I can close if it isn't happening. NBD, I am using a forked version of createjs to do those extra things that are missing. Would be willing to commit those extras too but obviously I don't want to put in the effort if you are not interested.

**Comment by lannymcnie** [comment]
Hey, sorry for the radio silence. 

Definitely still open to this PR. Unfortunately we have been pretty busy with production work, and haven't had much time to review stuff like this for CreateJS. When it comes to adding helper functions, we take a pretty conservative approach, but I think this sort of thing is valuable, so it is still on the table :)

Hopefully pushing out a pretty big update in the near future!

**Comment by jschatz1** [comment]
Thanks @lannymcnie. I totally understand! 😄

**Comment by jschatz1** [comment]
Thank you so much @lannymcnie!

**Pull request merged**